sql/knapsack_0-1/knapsack_0-1.sql

```diff
@@ -0,0 +1,51 @@
+/*PGR-GNU*****************************************************************
+File: _knapsack_0-1.sql
+
+Copyright (c) 2021 pgRouting developers
+Mail: project@pgrouting.org
+
+Function's developer:
+Copyright (c) 2021 Celia Virginia Vergara Castillo
+Copyright (c) 2021 Joseph Emile Honour Percival
+
```

Comment:
I do not see you name as the function developer.
Ar you going to give Iosefa and me the credit for your work?